### PR TITLE
add workflow for release build with image-builder

### PR DESCRIPTION
**Pull request by kolodziejczak**
**Description**

Changes proposed in this pull request:

- add workflow for release build with image-builder

### CODE REVIEW

File: .github/workflows/create-release.yaml [hunk 1]
```diff
@@ -85,29 +87,26 @@ jobs:
...
-        name: Verify build of experimental image
-        id: wait-for-experimental-build
+          name: istio-manager
+          dockerfile: Dockerfile
+          context: .
+          build-args: |
+            VERSION=${{ github.event.inputs.name }}
+          tags: "${{ github.event.inputs.name }}"
+
+      - name: Image build - experimental
```
Comment (triffer) [inline .github/workflows/create-release.yaml:100]: Should we use the same name we use on main?
```suggestion
      - name: Build manager image - experimental
```

File: .github/workflows/create-release.yaml [hunk 2]
```diff
@@ -85,29 +87,26 @@ jobs:
         with:
           fetch-depth: 0
 
-      - uses: autotelic/action-wait-for-status-check@6556cf50c8fb6608412945382eae73581f56cbb4
-        name: Build image
-        id: wait-for-build
+      - name: Image build
```
Comment (triffer) [inline .github/workflows/create-release.yaml:90]: Should we use the same name we use on main?
```suggestion
      - name: Build manager image
```

Comment (kolodziejczak) [reply .github/workflows/create-release.yaml:90]: looks better, let's do that

File: .github/workflows/create-release.yaml (hunk 1 above)
Comment (kolodziejczak) [reply .github/workflows/create-release.yaml:100]: looks better, let's do that

**Review by triffer — APPROVED** [review]

**Pull request merged**